src/components/atoms/UserAvatar/UserAvatar.scss

```diff
@@ -1,23 +1,34 @@
 @import "scss/constants";
 
 .UserAvatar {
-  $avatar-sizes-map: (
-    small: 25px,
-    medium: 40px,
-    large: 54px,
-    full: 100%,
-  );
-  $indicator-sizes-map: (
-    small: 8px,
-    medium: 10px,
-    large: 12px,
-    full: 25%,
-  );
-
-  // NOTE: parent and img mismatch due to legacy reasons
+  // @see https://math.stackexchange.com/questions/377234/circle-in-square-calculate-distance-from-squares-corner-to-circles-perimeter
+  $square-root-of-2: 1.41421356237;
+  $indicator-offset-factor: ($square-root-of-2 - 1) / 2;
+  $indicator-radius-factor: $square-root-of-2 / 2;
+
+  // indicator border should be 12% of the indicator size, but borders don't accept % so using px for default
+  $default-indicator-border-width: 2px;
+  $default-indicator-size: 28%;
```

Comment:
Is there a reason for these consts to be inside the class instead of being at the top of the file?

Reply:
Scoping. 

They are relevant only to that component so the `.UserAvatar {}` block envelopes everything that should be only part of its scope. 

Note: they're still technically "at the top" of the file, since there are only 4 lines ahead.

Reply:
We don't need scoping for the scss variables as they are complied so please move them to the top as the rest of the codebase

Reply:
Since the scss variables are not complied into the css and are processed on the module level, please just move them to the top of the file to match the code style of the rest of the codebase

Reply:
I will move them to be consistent with the style even though module scope isn't required.

Reply:
done f70c61dd2eaf3b9d9db27d41da9590ab95cb1fde